### DISCUSSION TITLE
How to tell chezmoi to accept changes in the destination directory without an explicit merge?

### DISCUSSION THREAD

**Discussion by halloleo** [twpayne/chezmoi#3902] [Q&A] [answered]
I am tracking my dot files for a while with chezmoi. This works great -- particularly to trickle down changes from the source directory to HOME, i.e. the destination directory.

But I have a question about accepting changes in the other direction: Say, I am tracking  my `~/.bash_profile` already with chezmoi. Now I have changes in the destination file `~/.bash_profile` and I know I want to accept _all_ changes in this file.

At the moment t I use the merge command `cz merge ~/.bash_profile` and add the changes via the UI of my merge tool (VS Code) into the source  directory. 

But is there an easier way to do this without the round trip via my merge tool, just via chezmoi's commands? As said, I know (from diffing) that _all_ changes from the file `~/.bash_profile` are to be accepted.

Thanks for any help!

**Comment by twpayne** [comment, marked as answer]
Use `add` again, i.e.

    chezmoi add ~/.bash_profile

**Reply by halloleo** [reply]
Thanks for chiming in.

I did so, but `chezmoi diff ~/.bash_profile` still lists the file as with differences.

**Reply by twpayne** [reply]
What is the exact output of `chezmoi diff`? What are the contents of `~/.bash_profile` and `~/.local/share/chezmoi/dot_bash_profile`?

**Reply by halloleo** [reply]
I have done the accepting this time via merge, but I will keep teh details next time if this comes up again.

However, _one_ thing I do know is that the `chezmoi add ~/.bash_profile` command outputted the following warning

```
warning: in the working copy of 'dot_bash_profile', LF will be replaced by CRLF the next time Git touches it
```

**Reply by twpayne** [reply]
This warning comes from git, not chezmoi. I guess you're on Windows. You need to make sure your git `core.autocrlf` configuration variable is set correctly.
